### PR TITLE
[MIG] migrate business_requirement_deliverable_resource_template from 10.0 to 12.0

### PR DESCRIPTION
**Pull request by ruter-lyu**

### CODE REVIEW

**Review by elicoidal — COMMENTED** [review]
LGTM. Please prepare the  README

**Comment by elicoidal** [comment]
@JoJoJoJoJoJoJo @joycehuan @seb-elico please have a look

File: business_requirement_deliverable_resource_template/README.rst [hunk 1]
```diff
@@ -0,0 +1,101 @@
+.. figure:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+
+====================
+Business Requirement
```
Comment (JoJoJoJoJoJoJo) [inline business_requirement_deliverable_resource_template/README.rst:7]: Change the name to module name.

File: business_requirement_deliverable_resource_template/models/business_requirement_resource_template.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    _inherit = "business.requirement.deliverable"
+
+    def _prepare_resource_lines(self):
+        rl_data = self.env['business.requirement.resource.template'].search(
+            [('product_template_id', '=', self.product_id.product_tmpl_id.id)],
+            order='sequence')
+        data = []
+        for rec in rl_data:
+            data.append(rec.copy_data()[0])
+        return [(0, 0, item) for index, item in enumerate(data)]
```
Comment (JoJoJoJoJoJoJo) [inline business_requirement_deliverable_resource_template/models/business_requirement_resource_template.py:17]:
```suggestion
        return [(0, 0, item) for item in data]
```

**Review by JoJoJoJoJoJoJo — COMMENTED** [review]
Some improvement.

File: business_requirement_deliverable_resource_template/README.rst [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+#. Adapt the resources lines if necessary
+
+.. figure:: static/img/bus_req_default2.png
+   :width: 600 px
+   :alt: The default resource lines are automatically added to your BR
+
+.. figure:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/222/10.0
```
Comment (nikul-serpentcs) [inline business_requirement_deliverable_resource_template/README.rst:61]: @ruter-lyu 10.0/12.0

File: business_requirement_deliverable_resource_template/README.rst [hunk 4]
```diff
@@ -0,0 +1,101 @@
+.. figure:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```
Comment (nikul-serpentcs) [inline business_requirement_deliverable_resource_template/README.rst:3]: @ruter-lyu LGPL-3.0

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
netpicking

**Comment by pedrobaeza** [comment]
Closing as old and not green

**Pull request closed without merging**